### ISSUE TITLE
Fine tuning TFHub models

### ISSUE THREAD

**Issue by kkkppp** [google-research/google-research#139]
I tried V2 first, it doesn't work (with TF 1.15), there is an earlier report about missing Einsum op.

I tried V1 then for Cola task, after training it shows 

eval_accuracy = 0.6912752
eval_loss = 0.6295214
global_step = 6413
loss = 0.62965536

However if I run prediction, it always predicts label "1", resulting test file looks really strange:

0.2420986	0.7579014
0.24209863	0.7579013
0.24209863	0.7579013
0.24209863	0.7579013
...

So did anyone tried those TFHub models? They don't look like usual bert ones (no .ckpt files), so probably some conversion/preparation is required?
